src/forms/fields.js

```diff
@@ -43,6 +42,19 @@ function isIntegerValidator(value) {
 }
 isIntegerValidator.message = 'number_should_not_have_decimals';
 
+function isEmpty(value) {
+    return typeof value === 'undefined' || value === null || value === ''
+}
```

Comment:
Should be able to use `isEmpty` `lodash/fp`, as suggestion above?

Reply:
I'm going to merge as is because this code is already in master branch. I took the code for the validator from dhis2/ui (where there's no lodash dependency and isEmpty is a helper function: https://github.com/dhis2/ui/blob/master/collections/forms/src/validators/helpers/index.js#L2).